daemon/emer-machine-id-provider.c

```diff
@@ -186,9 +187,18 @@ emer_machine_id_provider_class_init (EmerMachineIdProviderClass *klass)
 static void
 emer_machine_id_provider_init (EmerMachineIdProvider *self)
 {
+  g_autoptr(GError) local_error = NULL;
   EmerMachineIdProviderPrivate *priv =
     emer_machine_id_provider_get_instance_private (self);
 
+
+  if (!write_tracking_id_file (TRACKING_ID_OVERRIDE, &local_error))
+    g_message ("Failed to initialize override machine id at %s: %s."
+               "Default machine id at %s will be used",
```

Comment:
I also don't think we want this behaviour. If you look at `read_machine_id`, it returns `FALSE` if the machine ID can't be read.

I suggest removing all code that accesses `/etc/machine-id`.

Reply:
> I suggest removing all code that accesses /etc/machine-id.

Ah yeah, I was going to ask you that. Great.